src/pathfora.js

```diff
@@ -1750,23 +1749,32 @@
         }
       };
 
-      var contentCb = function (content, w) {
-        if (content) {
-          w.content = {
-            0: {
-              title: content.title,
-              description: content.description,
-              url: 'http://' + content.url,
-              image: content.primary_image
+      var recContent = function (w) {
+        pathfora.addCallback(function () {
+          if (pathfora.acctid === '') {
+            pathfora.acctid = context.lio.account.id;
+          }
+
+          api.recommendContent(pathfora.acctid, w.recommend.ql.raw, function (resp) {
```

Comment:
not sure if this is the right place for it but some commenting on what exactly will happen / how content recos might be useful (e.g. how defaults work, how many are pulled in, etc)

treat it like another dev is looking through the code outside of lytics to understand whats going on.